README.md

```diff
@@ -861,13 +861,15 @@ registration: {
 ## Registration
 
 > **:warning: Beta feature:** The registration feature is currently a [Beta feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#beta). This widget functionality won't work unless your Okta organization is part of the Beta program. For help, contact support@okta.com.
+> **:warning: Widget version requirements:** The registration feature requires widget version 2.7.0 or older.
 
 To add registration into your application, configure your Okta admin settings to allow users to self register into your app. Then, set `features.registration` in the widget. You can add additional configs under the registration key on the [`OktaSignIn`](#new-oktasigninconfig) object.
 
 ```javascript
     var signIn = new OktaSignIn({
       baseUrl: 'https://acme.okta.com',
-      clientId: '{{myClientId}}', // REQUIRED
+      // clientId is not used with widget version 2.8.0 or higher. You have to setup Self Service Registration policies.
```

Comment:
Maybe we can just say `clientId is not required if you are using version 2.8.0 or higher of the widget.`. Not sure how useful this line is `You have to setup Self Service Registration policies.` because we are not telling them how to configure Self Service Registration policies.

Reply:
if we say clientId is not required... it will suggest clientId is optional parameter - which is not the case. During discussion with @hor-kanchan-okta we agreed to add the second sentence to indicate developers that there is a path forward with registration using self service registration policies.

Reply:
We can be explicit and say `clientId is not required if you trying to set up registration using version 2.8.0 or higher of the widget` 

The second sentence only says `You have to setup Self Service Registration policies.`IMO it  does not give them any information on how to setup Self Service Registration policies. Maybe we can add `For help, contact support@okta.com` ?

Reply:
Updated comment.. tweaked it a bit as 'if you are trying to setup registration' seemed redundant.